src/components/pages/HomePage/time.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * Inspired by James Tauber's Mars Clock
+ * https://github.com/jtauber/mars-clock
+ */
+
+// Difference between TAI and UTC. This value should be
+// updated each time the IERS announces a leap second.
+var tai_offset = 37;
+
+// Last time the above value changed.
+var last_leap_second = "1 January 2017";
+
+function cos(deg) {

```

Comment:
You could add a new line before here to improve readability.

Reply:
You should really start to use suggestions.
It is generally a cool feature. xD